dg/templates/social_website/includes/footer.html

```diff
@@ -43,12 +43,12 @@ <h3 class="hdg-a hdg-source-lt">Follow Us</h3>
         <!-- JAVASCRIPT -->
         
         <!-- For Debugging -->
-        <!--script data-main="{{STATIC_URL}}social_website/scripts/main" src="{{STATIC_URL}}social_website/scripts/libs/external/requirejs/require.js"></script-->
+        <script data-main="{{STATIC_URL}}social_website/scripts/main" src="{{STATIC_URL}}social_website/scripts/libs/external/requirejs/require.js"></script>
         <!-- Also change build to scripts in main.js -->
         
         <!-- For Production -->
-        <script type="text/javascript" src="{{STATIC_URL}}social_website/build/libs/external/requirejs/require.js"></script>
-        <script type="text/javascript" src="{{STATIC_URL}}social_website/build/main.js"></script>
+        <!--script type="text/javascript" src="{{STATIC_URL}}social_website/build/libs/external/requirejs/require.js"></script>
```

Comment:
Compile JavaScript 

New JavaScript Controller has to be added in Grunt File
And then discard changes in this file